dkist/dataset/tests/test_tiled_dataset.py

```diff
@@ -59,3 +61,18 @@ def test_tiled_dataset_from_components(dataset):
         assert ds.files == fm
         assert ds.meta["inventory"] is inventory
         assert ds.meta["headers"] is headers
+
+
+# @pytest.mark.mpl_image_compare
```

Comment:
```suggestion
```

Reply:
I thought this was redundant with `@figure_test`?

Reply:
Never mind I misread the suggestion. As you were.